packages/components-react/src/examples/PinCodeExampleControlled.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useState } from 'react';
+import type { PinCodeUpdateEvent } from '@porsche-design-system/components-react';
+import { PPinCode, PText } from '@porsche-design-system/components-react';
+
+export const PinCodeExampleControlledPage = (): JSX.Element => {
+  const [currentValue, setCurrentValue] = useState('');
```

Comment:
Could be renamed to just `value` and `setValue` because the current brings no benefit.